client/my-sites/earn/home.tsx

```diff
@@ -24,6 +24,7 @@ import {
 	FEATURE_SIMPLE_PAYMENTS,
 	FEATURE_UPLOAD_PLUGINS,
 } from 'lib/plans/constants';
+import analytics from 'lib/analytics';
```

Comment:
Using `lib/analytics` directly in components is discouraged. It's preferred to dispatch analytics actions via Redux.

Some details here: p4TIVU-38R-p2, I can provide examples from Calypso if that's helpful.

Reply:
Thanks. Updated.